rubin_sim/maf/plots/xyplotter.py

```diff
@@ -0,0 +1,31 @@
+import matplotlib.pyplot as plt
+from .plotHandler import BasePlotter
+
+
+__all__ = ["XyPlotter"]
+
+
+class XyPlotter(BasePlotter):
+    """Bare-bones plotter for making scatter plots."""
+
+    def __init__(self):
+        self.objectPlotter = True
+        self.plotType = "simple"
+        self.defaultPlotDict = {"title": None, "xlabel": "", "ylabel": ""}
+
+    def __call__(self, metricValueIn, slicer, userPlotDict, fignum=None):
+
+        plotDict = {}
+        plotDict.update(self.defaultPlotDict)
+        plotDict.update(userPlotDict)
+        plotDict.update(metricValueIn[0]["plotDict"])
+
+        fig = plt.figure(fignum)
+        ax = fig.add_subplot(111)
+        x = metricValueIn[0]["x"]
+        y = metricValueIn[0]["y"]
+        ax.plot(x, y)
+        ax.set_title(plotDict["title"])
+        ax.set_xlabel(plotDict["xlabel"])
+        ax.set_ylabel(plotDict["ylabel"])
+        return fig.number
```

Comment:
This doesn't seem unreasonable, but you could also make "x" and "y" user-configurable to plot any two metric values in a dictionary or recarray against each other.

Reply:
A plotter only gets one metric value at a time though, so I'm not sure how/why you would plot different metric values against each other.

Reply:
I wondered if my comment would sound like that and yes - I get that, but what I meant was other metrics which are returning more than one value (most of which have reduce functions) .. but could find it useful to plot their various values against each other.  Obviously nobody's doing that now, but it's not unreasonable .. like if you calculated season length and season cadence in one metric returning them both, then you could see how they were correlated in one step. 
Will this plotter fail if there's more than one data point in metricValues?

Reply:
Anyway - this is fine. I think if the plotter will only work if there's only one data point, then it's worth making a comment that this is so in the docstring, and making it configurable later is reasonable. 
(one note is that if you did make the x/y values user-configurable to be the names of what the metric returned, then those would also be reasonable to take as the labels for those axes).